frontend/src/components/AboutModal.tsx

```diff
@@ -97,7 +97,10 @@ export const AboutModal: FC = () => {
                   </Badge>
                   <Box mb={2}>A new version is available</Box>
                   <Box mb={2} fontStyle="italic" opacity={0.5}>
-                    Very soon you'll be able to trigger the update from here
+                    Very soon you'll be able to trigger the update from here.
+                  </Box>
+                  <Box mb={2} fontStyle="italic" opacity={0.5}>
+                    Until this time, you can download the latest version from the <u><a href="https://github.com/olup/lunii-admin/releases/latest">github repo</a></u>.
```

Comment:
Links like this will open in the app frame. There is a way to have external links  - check wails docs or I think I have an example somewhere in this repo

Reply:
https://github.com/olup/lunii-admin/blob/84f0f782e7622a3c3b6d38a1ea3ec62393e44c2e/frontend/src/components/NewPackModal.tsx#L55